### ISSUE TITLE
Discussion - new landmarks, alt text, modal

### ISSUE THREAD

**Issue by daironreijna** [daironreijna/Landmark-Learning-API#34]
1. Change locations/landmarks that are being used on the cards. 
2. Add missing ALT Text to Cards
3. Consider the next steps due to Google API only being USA-based.
4. Idea to use a modal instead of the hide/show option.
